puppet/zulip/manifests/base.pp

```diff
@@ -18,6 +18,8 @@
         '20.04' => 'focal',
       }
       $base_packages = [
+        # systemctl
+        'systemd',
```

Comment:
@andersk can you look at this?  I have this recollection that some Docker images break use of systemd, but don't remember the details.

Reply:
Yeah it’s not just some Docker images; Docker and systemd [just plain hate each other](https://developers.redhat.com/blog/2016/09/13/running-systemd-in-a-non-privileged-container/). Systemd refuses to run in an unprivileged Docker container without a bunch of sketchy manual twiddling on the host. Thus, docker-zulip runs supervisord as PID 1. So I would be surprised if a reasonable-looking change like this works there out of the box.

Reply:
Ugh.  OK, good to know!  I just re-pushed using the existing duality of systemd on redhat and sysv-init on debian/ubuntu.  Theoretically, given what you say, if you tried to run redhat in docker, it'd also explode -- but we can kick that down the road a little.